src/engraving/libmscore/measure.cpp

```diff
@@ -3746,7 +3746,6 @@ void Measure::addSystemHeader(bool isFirstSystem)
                 clef->layout();
             } else if (clef) {
                 clef->parentItem()->remove(clef);
-                delete clef;
```

Comment:
As far as I remember, we cannot remove generated elements, so they don't go to UndoStack. Can we rewrite this code as follows:

```
if (clef->generated()) {
    delete clef;
}
```

Will this fix the crash?

Reply:
Thanks, that solution works too!